react/components/molecules/card/card.js

```diff
@@ -0,0 +1,199 @@
+        };
+    }
+
+    _iconStyle() {
+        return {
+            ...styles.avatar,
+            ...this.props.style?.icon
+        };
+    }
+
+    _textStyle() {
+        return {
+            ...styles.text,
+            ...this.props.style?.text
+        };
+    }
+
+    _renderCardImageAvatar() {
+        return (
+            <Avatar
+                image={{
+                    uri: this.props.avatarURL
+                }}
+                size={60}
+                style={this._avatarStyle()}
+            />
+        );
+    }
+
+    _renderCardImageIcon() {
+        return (
+            <Icon
+                style={this._iconStyle()}
+                icon={this.props.icon}
+                color={this.props.iconColor}
+                fill={this.props.iconFill}
+                height={60}
+                width={60}
+            />
+        );
+    }
+
+    _renderCardImage() {
+        if (this.props.icon) return this._renderCardImageIcon();
+        if (this.props.avatarURL) return this._renderCardImageAvatar();
+    }
+
+    _renderCardText() {
+        return (
+            <View style={styles.textContainer}>
+                {this.props.title ? (
+                    <Text numberOfLines={1} style={styles.title}>
+                        {this.props.title}
+                    </Text>
+                ) : null}
+                {this.props.text ? (
+                    <Text numberOfLines={2} style={styles.text}>
+                        {this.props.text}
+                    </Text>
+                ) : null}
+                {this.props.subtext ? (
+                    <Text numberOfLines={1} style={styles.text}>
+                        {this.props.subtext}
+                    </Text>
+                ) : null}
+            </View>
+        );
+    }
+
+    render() {
+        return (
+            <Item
+                style={this._style()}
+                contentStyle={this._contentStyle()}
+                variant={this.props.shapeVariant}
+                onPress={this.props.onPress}
+                disabled={this.props.disabled}
+                onLongPress={this.props.onLongPress}
+                activeOpacity={0.6}
+                {...this.props.itemProps}
+            >
+                {this.props.children ? (
+                    this.props.children
+                ) : (
+                    <>
+                        {this._renderCardImage()}
+                        {this._renderCardText()}
+                    </>
+                )}
+            </Item>
+        );
+    }
+}
+const styles = StyleSheet.create({
+    style: {
+        paddingHorizontal: 0
+    },
+    contentStyle: {
+        alignItems: "center",
+        backgroundColor: "#ffffff",
+        flexDirection: "row",
+        height: 100,
+        shadowColor: "#435764",
+        elevation: 5,
+        shadowOffset: { width: 0, height: 2 },
+        shadowOpacity: 0.4,
+        overflow: "hidden",

```

Comment:
This line is causing the shadow to not appear in iOS
<img width="387" alt="image" src="https://user-images.githubusercontent.com/25725586/125414445-6bd1a2ea-8a7f-47ae-b329-681d06f01772.png">